Alicerce/Sources/Logging/Log.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  Alicerce.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 03/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public final class Log: Logger {
+
+    public final class Queue {
```

Comment:
I think we should add a comment explaining the reason for this `Queue` (which is to force that injected queues on `LogDestination`'s are serial), don't you think? 😇